charts/redpanda/templates/_configmap.tpl

```diff
@@ -0,0 +1,463 @@
+{{/*
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/}}
+
+{{- define "configmap-content-no-seed" -}}
+{{- /*
+  configmap content without seed list.
+*/ -}}
+{{- $root := . }}
+{{- $values := .Values }}
+
+{{- /*
+  It's impossible to do a rolling upgrade from not-tls-enabled rpc to tls-enabled rpc.
```

Comment:
NIT: Interesting comment, is there work around to enable TLS on rpc?

Reply:
This is just me copy pasting the configmap details here. Do you want me to take a look at this?

Reply:
not as part of this PR

Reply:
> not as part of this PR

@joejulian my question still stand as I would like to know if there is a work around. 

P.S. The PR is already approved. 

P.S. 2 I moved to convo to the slack.